releases/borg-1.1.rst

```diff
@@ -0,0 +1,252 @@
+  This is enabled by default, see BORG_HOSTNAME_IS_UNIQUE.
+- Answers to prompts like "Accessing previously unknown repository" are now saved immediately.
+- Cache synchronization and "borg info" are now faster.
+- Reduced space usage of chunks.archive.d in the cache by 30-40 %. Existing caches are migrated during a cache sync.
+- The cache used for remote cache syncs and mounting remote repositories does not grow indefinitely any more,
+  but adapts to the available space. Good riddance, TMP=/var/tmp!
+- BORG_PASSCOMMAND makes using key rings and hardware keys much easier.
+
+Documentation improvements:
+
+- The online command reference has been significantly revamped:
+  better presentation of options, commands are now separate pages
+  (improves search, and old links still work), many formatting improvements,
+  revised examples, a new theme, ...
+- New deployment guides
+- The internals docs saw many corrections and extensions.
+- man pages are now provided in source releases.
+- ``--help`` uses ANSI sequence on terminals for basic markup
+- New asciinema screencasts.
+
+Compatibility notes for upgrading from Borg 1.0 to Borg 1.1:
+
+- **No explicit "borg upgrade" is required.**
+- Borg 1.1 uses some new data structures which are backwards-compatible with Borg 1.0.4 and newer.
+- Borg 1.0 can't make use of Borg 1.1's "compact cache" and will silently ignore it.
+- The default compression has been changed from "none" to "lz4".
+- Repositories in the "repokey" and "repokey-blake2" modes with an empty passphrase are now treated as
+  unencrypted repositories for security checks (e.g. BORG_UNKNOWN_UNENCRYPTED_REPO_ACCESS_IS_OK).
+- borg init:
+
+  - -e/--encryption is now a mandatory option with no default value.
+    Previously the default value was "repokey".
+  - the short form of --append-only (-a) has been removed.
+  - running "borg init" via a "borg serve --append-only" server will *not* create
+    an append-only repository any more. Use "borg init --append-only" to initialize an append-only repository.
+- borg create: the --exclude-if-present option now supports tagging a folder with any file system
+  object type (file, folder, etc.), instead of accepting only files as tags.
+- borg upgrade: the short form of --inplace (-i) has been removed.
+- borg delete: the short form of --cache-only (-c) has been removed.
+- "borg migrate-to-repokey" has been renamed to "borg key migrate-to-repokey" with no deprecation notice,
+  since it is rarely and only manually used.
+
+Deprecated options and commands in Borg 1.1, which will be removed in Borg 1.2:
+
+- "borg change-passphrase" is deprecated, use "borg key change-passphrase" instead.
+  "borg change-passphrase" will be removed in Borg 1.2.
+- borg create: the --keep-tag-files option has been deprecated in favour of the new --keep-exclude-tags option.
+  Both options have the same effect. --keep-tag-files will be removed in Borg 1.2.
```

Comment:
Need to add the --files-cache=... related deprecations here...

Reply:
- --ignore-inodes (use --files-cache modes without "inode")
- --no-files-cache (use --files-cache=disabled)